code/processes/rdb.q

```diff
@@ -88,7 +89,9 @@ endofday:{[date]
 			.lg.o[`endofday;"reload is enabled - storing counts of tables at EOD : ",.Q.s1 eodtabcount];
 			.lg.o[`endofday;"Escaping end of day function"];:()];
 	t:tables[`.] except ignorelist;
-	/-get all attributes from all tables before they are wiped
+	/-set eod attributes on gateway for rdb
+	gateh:exec w from .servers.getservers[`proctype;.rdb.gatewaytypes;()!();0b;0b];
+	neg[gateh]@\:(`setattributes;.proc.procname;.proc.proctype;.proc.getattributes[]);
 	/-get a list of pairs (tablename;columnname!attributes)
```

Comment:
I think you have to flush the handle here as well. I think you could use .async.send here instead. I think it would be

.async.send[0b;;(`setattributes;.proc.procname;.proc.proctype;.proc.getattributes[])] each gateh